kahuna/public/js/components/gr-panels/gr-panels.css

```diff
@@ -18,7 +18,6 @@
     width: 290px;
     position: fixed;
     background: rgba(68, 68, 68, 0.9);
-    overflow: auto;
```

Comment:
On long panels this may allow things to overflow out of the page. I'm not sure there's a good solution for this as you can't have it hidden in one direction and visible in the other. Just something someone may point out.